Spigot-Server-Patches/0696-Can-Fly-Fall-Damage.patch

```diff
@@ -0,0 +1,59 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TreyRuffy <TreyRuffy@users.noreply.github.com>
+Date: Sat, 13 Mar 2021 12:59:14 -0700
+Subject: [PATCH] Can Fly Fall Damage
+
+
+diff --git a/src/main/java/net/minecraft/server/EntityHuman.java b/src/main/java/net/minecraft/server/EntityHuman.java
+index cc2127b26e41182c14fa95afde878e9b5100a117..8e3176b39c20d7d13eed4b580522e3ca317e4b86 100644
+--- a/src/main/java/net/minecraft/server/EntityHuman.java
++++ b/src/main/java/net/minecraft/server/EntityHuman.java
+@@ -72,6 +72,7 @@ public abstract class EntityHuman extends EntityLiving {
+     public EntityFishingHook hookedFish;
+     // Paper start
+     public boolean affectsSpawning = true;
++    private boolean canFlyFallDamage = false;
```

Comment:
Based on the line above (`affectsSpawning`): this field should be public. That way one less setter is required. Breaks encapsulation, but we don't really care about that in NMS code.